### PR TITLE
HW01_Sean Zheng

### PR DESCRIPTION
**Pull request by zhengyinyuan**
(All comments below are detailed in the README.md)

The README file is modified and submitted following the [Markdown guide](https://guides.github.com/features/mastering-markdown/) in Github and the [workflow guide](http://cfss.uchicago.edu/hw00_homework_guidelines.html#homework_workflow) in the course website. They provides step-by-step instructions and are easy to follow. Simple texts are typed as in Word documents, while formatting and adding images/lists are achieved through the use of a few simple symbols.

A problem I encountered is to create indention in Markdown. I used the no-break space '&nbsp;' to do so because tabbing and manually adding spaces instead embed texts into a display dialog. I would love to learn a more efficient way for indention.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+- [x] _**Emphases**_
+- [x] List
+- [x] Image
+- [x] Links
+- [ ] A smart way to do indention (see __Reflections__)
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Following the [online markdown guide](https://guides.github.com/features/mastering-markdown/), I use markdown syntax to incorporate the elements listed in the [hw guide](http://cfss.uchicago.edu/hw01_edit-README.html). Markdown is easy to use where simple texts are typed as in a Word document. Complex functions such as including codes and outputs would be done through advanced R Markdown. 
+
+## Reflections
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The online guides and [workflow instruction](http://cfss.uchicago.edu/hw00_homework_guidelines.html#homework_workflow) are extremely helpful for step-by-step learning and submitting. One problem I had is to create separate paragraphs. It is not enough to only hit __ENTER__ once but instead twice, leaving an empty line inbetween two sentences in Rstudio.
```
Comment (srishtigoel) [inline README.md:29]: Interesting to note that!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
 # hw01
```
Comment (srishtigoel) [inline README.md:1]: Though this is a very petty issue but 'hw01' could have been edited, starting with a capital letter since this is a heading.

**Review by srishtigoel — COMMENTED** [review]
Evaluation HW 01 for Sean Zheng
coding style - excellent
coding strategy - excellent
presentation - excellent
achievement, creativity - excellent
ease of access - excellent

I would like to appreciate the use of reflections and the way Sean explained it at the end of the bio. It's good to explore things apart form the text. It was a learning for me too. Besides a very petty thing of not editing the 'hw01' title, I would say a great effort.

**Comment by jmausolf** [comment]
## Evaluation HW-01 for Sean Zheng

| Topic | Excellent | Satisfactory | Needs Work |
| --- | --- | --- | --- |
| **Coding style** |        x  |     |             |
| **Coding strategy** |     x   |        |        |
| **Presentation: graphs** |      |     |       |
| **Presentation: tables** |       |       |     |
| **Achievement, creativity** |x      |      |         |
| **Ease of access** | x |                 |             |

Remarks:
- For the header of the assignment, you probably could have removed the "hw01" header and subsequent lines with the assignment details. If you would like to include these, they could probably go in your pull request title/notes.
- Nice picture of a bear!
- For indents, you typically don't see them used in Markdown documents (at least not for indenting paragraphs). Your method seems to work. You can also use various list bullets to achieve indents of single lines. For PDF's you can get fancy with LaTex to achieve hanging indents, such as for citations:
```
\setlength{\parindent}{-0.2in}
\setlength{\leftskip}{0.2in}
\setlength{\parskip}{10pt}
```

*I haven't tried this for web content, however. Some LaTex, such as formula's can be used for HTML in .Rmd.*
- Otherwise, you effectively used five of the five style elements, italics, photo, headers, list, and link. Good job.
- Great use of git. Proved you can do the workflow.

**Comment by banerjeeesha** [comment]
Peer Review for Sean Zheng
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       +    |              |            |
| **Coding strategy**         |    +       |              |            |
| **Presentation**            |        +   |              |            |
| **Achievement, creativity** |    +       |              |            |
| **Ease of access**          |      +     |              |            |

Remarks:

* All the requirements were met.
* I liked the usage of emojis, indention. I especially liked the usage of list with check marks under report section of the Readme.md. This is something I did not know.

**Pull request closed without merging**